OWML.SampleMods/OWML.LoadCustomAssets/default-config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "enabled": true,
+  "enabled": false,
```

Comment:
hehe, sure ;)

Reply:
Should I keep it? I did it mostly because of nullrefs

Reply:
Oh you still get those? I thought I fixed them. :(

Reply:
no, I mean that I used to get nullrefs, got tired of disabling manually and disabled it in config, than kinda forgot about it

Reply:
Ok, check now :-)